docs/atomic-uninstall.1.md

```diff
@@ -49,7 +49,7 @@ The uninstall command will not execute if --display is specified.
 If --display is not specified the uninstall command will execute.
 
 **-f** **--force**
-  Remove all containers based on this image
+  Remove all containers based on this image.  Uninstall will fail if `--force` is not used and there are still containers using the image.
```

Comment:
Add something to the description saying that the image will not be removed if there are any containers using the image.